src/main/scala/com/cloudera/sparkts/DateTimeIndex.scala

```diff
@@ -105,6 +104,13 @@ trait DateTimeIndex extends Serializable {
   def locAtDateTime(dt: Long): Int
 
   /**
+   * The location of the given date-time. If the index contains the date-time more than once,
+   * returns its first appearance. If the given date-time does not appear in the index, returns the
+   * location of the closest anterior DateTime that exists.
+   */
+  def locAtOrBeforeDateTime(dt: ZonedDateTime): Int
```

Comment:
True, that was a local change for the irregular lagging / toUniform issue, and I forgot to remove it before committing. Will do that.